packages/dev/gui/src/2D/controls/stackPanel.ts

```diff
@@ -236,6 +224,42 @@ export class StackPanel extends Container {
         super._postMeasure();
     }
 
+    public isWidthFullyDefined(): boolean {
+        if (!this._isDirty && this._cachedIsWidthFullyDefined !== undefined) {

```

Comment:
Same comment than above, I'm not sure if updating a child control will set the `_isDirty` flag of the parent `StackPanel` to `true`.